tools/experiments/experiments.js

```diff
@@ -348,7 +388,7 @@ function getAmpConfig() {
   xhr.send(null);
   return promise
     .then(text => {
-      const match = text.match(/self\.AMP_CONFIG=([^;]+)/);
+      const match = text.match(/self\.AMP_CONFIG=({.+?})/);
```

Comment:
What is this syntax?

Reply:
It's a mistake 🙃 it should've been `(\{.+?\})` - fixed.

If you're asking what's `.+?` it means any character (`.`) repeated one or more times (`.+`), but as few matches as possible (`.+?`) to continue to match the rest of the expression. Basically:

```javascript
const str = 'abcd_ efgh_';
str.match(/(.+)_/);  // entire match is 'abcd_ efgh_' and group 1 is 'abcd_ efgh'
str.match(/(.+?)_/); // entire match is 'abcd_' and group 1 will be 'abcd'
```

Reply:
LOL nope I was asking about the braces haha